pennylane/transforms/qcut.py

```diff
@@ -827,6 +827,118 @@ def expand_fragment_tapes_mc(
     return all_configs, settings
 
 
+def _reshape_results(results: Sequence, communication_graph: MultiDiGraph, shots: int):
+    """
+    Helper function to reshape results and find out degrees of communication
+    graph
+    """
+    results = [qml.math.flatten(r) for r in results]
+    results = np.array(results, dtype=object)
+    results = results.reshape((len(communication_graph), shots)).T
+
+    return results
+
+
+def qcut_processing_fn_sample(
+    results: Sequence, communication_graph: MultiDiGraph, shots: int
+) -> List[np.array]:
```

Comment:
```suggestion
) -> List:
```
This could be e.g. a TF tensor